main/inputoutput/io.py

```diff
@@ -1,2 +1,9 @@
+from zipfile import ZipFile
 def inc(num):
-    return num+2
\ No newline at end of file
+    return num+2
+
+def unzip_file(fromPath,toPath):
+    file=ZipFile(fromPath,'r')
+    file.extractall(toPath)
+    file.close()
+    return 1
```

Comment:
Ideally we should not keep the heavy files in git repo. 
You can have a file under 256 KB for unzipped and the smaller zipped version of the same file. 

Also, try not to keep any valid data in the files. 
You can create some text files by copying some data from Wikipedia or other public knowledge websites.
If you feel fancy, you can try some online data generators to get you a csv file.